tensorflow/core/util/gpu_launch_config.h

```diff
@@ -180,12 +180,12 @@ inline GpuLaunchConfig GetGpuLaunchConfig(int work_element_count,
   //CHECK_EQ(err, hipSuccess);
 
   const int physical_thread_count = std::min(
```

Comment:
HIP doesn't support `hipOccupancyMaxPotentialBlockSize()` yet so codes here can't be merged